src/types/ErrorType.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorType {
+  LOAD = 'Unable to load tods',
```

Comment:
Fix the typo in the error message: 'Unable to load tods' should be 'Unable to load todos' 📝